### PR TITLE
Run zuul for pip and rpm

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Run zuul tests both on pip and rpm.

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -3,26 +3,46 @@
     name: packit-service/ogr
     check:
       jobs:
-        - tests
-        - pre-commit
+        - ogr-pre-commit
```
Comment (rpitonak) [inline .zuul.yaml]: wouldn't `pre-commit` be enough?

File: .zuul.yaml [hunk 2]
```diff
@@ -3,26 +3,46 @@
...
+        - ogr-tests-rpm
+        - ogr-tests-pip-deps
     gate:
       jobs:
-        - tests
-        - pre-commit
+        - ogr-pre-commit
```
Comment (rpitonak) [inline .zuul.yaml]: same as above

File: files/tasks/build-rpm-deps.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+- name: Install build RPM dependencies
```
Comment (rpitonak) [inline files/tasks/build-rpm-deps.yaml]: I would switch `RPM` with `build`

File: .zuul.yaml (hunk 1 above)
Comment (lachmanfrantisek) [reply .zuul.yaml]: The zuul needs identical job names accross all projects.

As you can see in the first comment by zull that is hidden:

```
Job tests-rpm in packit-service/packit is not permitted to shadow job tests-rpm in packit-service/ogr
```

So I added prefix to all jobs.

File: files/tasks/build-rpm-deps.yaml (hunk 3 above)
Comment (lachmanfrantisek) [reply files/tasks/build-rpm-deps.yaml]: I've copied that from packit but I'll change it.

File: .zuul.yaml (hunk 1 above)
Comment (rpitonak) [reply .zuul.yaml]: Ok, I see. So this is resolved then.

**Pull request merged**